rasa/core/evaluation/markers.py

```diff
@@ -0,0 +1,95 @@
+import os
+from pathlib import Path
+from typing import (
+    Dict,
+    Text,
+    Union,
+)
+
+import rasa.shared.core.constants
+from rasa.shared.exceptions import RasaException
+import rasa.shared.nlu.constants
+import rasa.shared.utils.validation
+import rasa.shared.utils.io
+import rasa.shared.utils.common
+from rasa.shared.data import is_likely_yaml_file
+from ruamel.yaml.parser import ParserError
+
+
+class InvalidMarkersConfig(RasaException):
+    """Exception that can be raised when markers config is not valid."""
+
+
+class MarkerConfig:
```

Comment:
Should this rather be a bunch of functions? As it doesn't even instantiate the class I don't see the purpose - it might even confuse readers as they expect an object but then it's just a dictionary

Reply:
Do classes in python always need to be instantiatable? Or is it a coding convention that we have? I guess I'm using classes to conceptually organise things. Maybe that's not desirable. 

For now we're representing the configs as dicts. If this remains the case, I'm happy to switch these into functions if that's preferred. If we end up needing a more elaborate representation then we might need the class. How does that sound?

Reply:
It's for sure not wrong but I'd very _uncommon_ (in lack of a better word I'd say it's not _pythonic_, e.g. comment [here](https://stackoverflow.com/questions/42757961/a-class-with-all-static-methods#comment72631648_42757961) or comment [here](https://stackoverflow.com/questions/11788195/module-function-vs-staticmethod-vs-classmethod-vs-no-decorators-which-idiom-is)). In cases like these you'd usually rather use modules to organize things than static methods.

In this particular case I'd say that the class is slightly confusing to the reader as they'd expect a class instance to be returned (`@classmethod` and things like `from_yaml` at Rasa are almost always used as constructors). I think the functions would make it more obvious that we are dealing with primitive data objects here.

Reply:
>In this particular case I'd say that the class is slightly confusing to the reader as they'd expect a class instance to be returned (@classmethod and things like from_yaml at Rasa are almost always used as constructors). I think the functions would make it more obvious that we are dealing with primitive data objects here.

Great point. Will refactor 👍